core/src/main/res/values-fr/strings.xml

```diff
@@ -156,81 +156,81 @@
+    <string name="settings_save_description">Avant de créer une sauvegarde, assurez-vous que la dernière version de l\'application est installée pour éviter tout conflit.</string>
+    <string name="settings_restore_backup">Restaurer une sauvegarde</string>
     <string name="settings_restore_description">Les données actuelles seront perdues.</string>
-    <string name="settings_dialog_message">Restaurer la sauvegarde?</string>
-    <string name="settings_export_csv">Exportation de feuille de calcul</string>
-    <string name="settings_export_csv_description">Exportez les enregistrements d\'activité sous forme de fichier csv pour les afficher dans votre application préférée de feuille de calcul.</string>
+    <string name="settings_dialog_message">Restaurer la sauvegarde ?</string>
+    <string name="settings_export_csv">Exporter dans une feuille de calcul</string>
+    <string name="settings_export_csv_description">Exportez les enregistrements d\'activité sous la forme de fichier CSV pour les afficher dans le tableur de votre choix.</string>
     <string name="settings_file_open_error">Erreur lors de l\'ouverture du fichier</string>
     <string name="settings_file_create_error">Erreur lors de la création du fichier</string>
 
-    <string name="message_backup_saved">Backup saved</string>
-    <string name="message_backup_restored">Backup restored</string>
-    <string name="message_save_error">Error saving backup</string>
-    <string name="message_restore_error">Error restoring backup</string>
-    <string name="message_app_not_found">Application not found</string>
-    <string name="message_csv_export_complete">Export completed</string>
-    <string name="message_csv_export_error">Error exporting csv</string>
+    <string name="message_backup_saved">Sauvegarde terminée</string>
+    <string name="message_backup_restored">Restauration terminée</string>
+    <string name="message_save_error">Erreur lors de la sauvegarde</string>
+    <string name="message_restore_error">Erreur lors de la restauration</string>
+    <string name="message_app_not_found">Application non trouvée</string>
+    <string name="message_csv_export_complete">Exportation en CSV terminée</string>
+    <string name="message_csv_export_error">Erreur lors de l\'exportation en CSV</string>
 
     <!--  Statistics detail  -->
     <string name="statistics_detail_chart_hourly">toutes les heures</string>
-    <string name="statistics_detail_chart_daily">quotidiennement</string>
-    <string name="statistics_detail_chart_weekly">hebdomadairement</string>
-    <string name="statistics_detail_chart_monthly">mensuellement</string>
-    <string name="statistics_detail_chart_yearly">annuellement</string>
+    <string name="statistics_detail_chart_daily">jour</string>
+    <string name="statistics_detail_chart_weekly">semaine</string>
+    <string name="statistics_detail_chart_monthly">mois</string>
+    <string name="statistics_detail_chart_yearly">année</string>
```

Comment:
Here is another change that might cause disagreement.

This change is related to the stats detail chart shown below (currently mostly un-translated).
I simply think it works best with shorter text.

<img width=400 src="https://user-images.githubusercontent.com/10946971/188885738-393b3a2b-4aa1-4c86-a1a3-42fcab2c6532.png" />

I believe it's easier to understand how to use the chart if the selected period is a period of time (a day, a week) and not a frequency (daily, weekly). If I guess right, the idea behind using daily/weekly/… is because the chart shows daily averages, but then the text `Average for period: daily` is confusing, and more confusing is the text `Without empty periods`.

However this change is basically opinion-based, and pretty minor, so I will gladly drop those changes from my PR if they are a source of disagreement.